inc/saimacsec.h

```diff
@@ -1273,13 +1292,13 @@ typedef struct _sai_macsec_api_t
     sai_remove_macsec_port_fn           remove_macsec_port;
     sai_set_macsec_port_attribute_fn    set_macsec_port_attribute;
     sai_get_macsec_port_attribute_fn    get_macsec_port_attribute;
-    sai_get_macsec_port_stats_fn        get_macsec_port_stats;
```

Comment:
Any considerations for the backward compatibility. Cant these be retained along with finer granularity flow stats.

Reply:
This is good point we can keep port stats function for backward compatibility.